### PR TITLE
fix formatting in api.php, data.php and queries.php

### PR DESCRIPTION
**Pull request by 00firestar00**
I'll start off with an easy PR.
Converts tabs to spaces, removes trailing whitespace and fixes improper indentation 

@pi-hole/dashboard

### CODE REVIEW

File: queries.php [hunk 1]
```diff
@@ -51,14 +51,14 @@
...
-        tableApi = $('#all-queries').DataTable( {
-            "rowCallback": function( row, data, index ){
-            	if (data[4] == "Pi-holed") {
-            		$(row).css('color','red')
-            	}
-            	else{
-            		$(row).css('color','green')
-            	}
+        tableApi = $('#all-queries').DataTable({
+            "rowCallback": function (row, data, index ) {
```
Comment (Mcat12) [inline queries.php]: Just personal preference, but can you remove the space between function and the parentheses? (Same for for loops and such)

**Comment by 00firestar00** [comment]
As an alternative to merging this PR. I think it would be better to come up with a unified code style that can be placed in the contributing readme.

It seems like every person that is touching these files has a different preference. And from my experience, having a consistent coding style makes things much easier to read/manage
for example:
```
foreach($qs as $q){
```
VS
```
foreach ($qs as $q) {
```
VS
```
foreach ( $qs as $q ) {
```

**Comment by Mcat12** [comment]
Why not both? :) I agree, we need to have some coding standards. Perhaps with the (unconfirmed, but possible) change to Python standards will be easier to create and enforce.

**Comment by Mcat12** [comment]
We'll look into setting coding standards (when we get free time. Finals are coming...).

**Comment by Mcat12** [comment]
Closing PR, since it's way out of date. The Python rewrite will have the PEP8 code styling, and the web interface style will be looked into once we see what we're going to be making it in.

**Pull request closed without merging**